Source/Core/DolphinQt/GameList/GameListModel.cpp

```diff
@@ -187,6 +188,22 @@ QVariant GameListModel::data(const QModelIndex& index, int role) const
       return compression.isEmpty() ? tr("No Compression") : compression;
     }
     break;
+  case Column::TimePlayed:
+    if (role == Qt::DisplayRole || role == SORT_ROLE)
+    {
+      TimePlayed timer(game.GetGameID());
+
+      //milliseconds to minutes conversion
+      int game_milliseconds = timer.GetTimePlayed();
+      int game_minutes = game_milliseconds / (60000);
+      int game_hours = game_minutes / 60;
+      int remaining_game_minutes = game_minutes % 60;
+
+      QString formatted_time = QString::number(game_hours) + tr("h, ") +
+                               QString::number(remaining_game_minutes) + tr("m");
```

Comment:
This can be very difficult to translate correctly, since the translator won't be able to reorder the different parts of the text. Please do it like this instead:

```suggestion
      // i18n: A time displayed as hours and minutes
      QString formatted_time = tr("%0h %1m").arg(game_hours, remaining_game_minutes);
```

I also added an i18n comment in case translators don't realize what h and m mean.